docs/source/en/tutorials/secure_code_execution.mdx

```diff
@@ -43,9 +43,17 @@ This is why we put emphasis on proposing code agents, in this case python agents
 By default, the `CodeAgent` runs LLM-generated code in your environment.
 
 This is inherently risky, LLM-generated code could be harmful to your environment.
+
+Malicious code execution could happen in at least 3 ways.
+- **Plain LLM error:** LLMs are still quite dumb, and they might destroy your system by trying to help you. This risk is low, but in my thousands of agent runs I've observed a few attempts to run code that could have been harmful.
+- **Supply chain attack** where you run a malicious LLM: then this LLM will generate code that actively tries to harm your system. This risk is extremely low when using well-known models on secure inference, but it could happen.
+- **Prompt injection:** an agent browsing the web could arrive on a malicious website that contains harmful instructions, thus injecting an attack into the agent's memory
+
+Once malicious code can be executed, accidentally or intentionally, it can harm your file system, abuse resources, be it local resources or API services, even compromise your network.
```

Comment:
Nit:
```suggestion
Once malicious code is executed, whether accidentally or intentionally, it can damage the file system, exploit local or cloud-based resources, abuse API services, and even compromise network security.
```